ray-operator/config/samples/vllm/ray-service.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: ray.io/v1
+kind: RayService
+metadata:
+  name: llama-3-8b
+spec:
+  serveConfigV2: |
+    applications:
+    - name: llm
+      route_prefix: /
+      import_path:  ray-operator.config.samples.vllm.serve:model
+      deployments:
+      - name: VLLMDeployment
+        num_replicas: 1
+        ray_actor_options:
+          num_cpus: 8
+          # NOTE: num_gpus is set automatically based on TENSOR_PARALLELISM
+      runtime_env:
+        working_dir: "https://github.com/ray-project/kuberay/archive/master.zip"
+        pip: ["vllm==0.4.3"]
+        env_vars:
+          MODEL_ID: "meta-llama/Meta-Llama-3-8B-Instruct"
+          TENSOR_PARALLELISM: "2"
+  rayClusterConfig:
+    headGroupSpec:
```

Comment:
I am not sure about the setup of our example document. Add the taint to avoid scheduling the head Pod on a GPU node if needed.

Reply:
Do you mean add a toleration to the worker group with GPUs? I don't think we usually apply taints for the head pod, the taint is for the nodes.

Reply:
I'll add this toleration to worker group for consistency with other GPU examples:
```
          # Please add the following taints to the GPU node.
          tolerations:
            - key: "ray.io/node-type"
              operator: "Equal"
              value: "worker"
              effect: "NoSchedule"
```

Reply:
How about using:

```yaml
tolerations:
- key: "nvidia.com/gpu"
  operator: "Exists"
  effect: "NoSchedule"
```

instead? This can maintain consistency with the existing YAML files.

Reply:
done